templates/corporate/upgrade.html

```diff
@@ -247,7 +264,11 @@ <h4>Describe your organization briefly</h4>
                                 <textarea name="description" style="width: 100%;" cols="100" rows="5" required></textarea>
                                 <br />
                                 <p id="sponsorship-discount-details"></p>
-                                <button type="submit" id="sponsorship-button" class="stripe-button-el invoice-button">Submit</button>
+                                <div class="upgrade-button-container" {% if is_demo_org %}data-tippy-content="{% trans %}Convert demo organization before upgrading.{% endtrans %}"{% endif %}>
```

Comment:
Why do we need this new container div?

Reply:
Because disabled elements do not fire any events and thus `tippyjs` will not work with disabled elements. Will add a comment explaining that.

Reply:
@amanagr @sahil839 -- are you aware of a cleaner way to handle giving Tippy tooltips to disabled elements?  I'm OK if this is necessary, but I feel like it's a pattern we have in settings and want to make sure we're not missing something.

Reply:
I am not sure if there are much cleaner ways.